packages/react/src/Tooltip/Tooltip.features.stories.tsx

```diff
@@ -8,7 +8,7 @@ import {SearchIcon} from '@primer/octicons-react'
 /* Tooltip v1 */
 
 export default {
-  title: 'Components/Tooltip/Features',
+  title: 'Deprecated/Components/TooltipV1/Features',
```

Comment:
Sorry for making a comment on the draft PR, let me know if this is just for testing etc. - Just wanted to mentioned that  we probably don't want to update the name from `Tooltip` to `TooltipV1` because that might break the [story name generation](https://github.com/primer/react/blob/main/packages/react/script/components-json/build.ts#L62) that we end up displaying on the docs site.

Reply:
Ah very interesting, good point. I'll swap it back. We can revisit naming and organization when we properly deprecate it.